src/Model/Forwarder.php

```diff
@@ -221,4 +223,40 @@ protected function createReviews($limit = null, $offset = null, array $uids = []
 
         return $reviews;
     }
+
+    /**
+     * @return array|mixed[]
+     */
+    public function createPluginInfo()
+    {
+        $objectManager   = ObjectManager::getInstance();
+        $productMetadata = $objectManager->get(ProductMetadataInterface::class);
+
+        return [
+            'Magento-Name'    => $productMetadata->getName(),
+            'Magento-Version' => $productMetadata->getVersion(),
+            'Magento-Edition' => $productMetadata->getEdition(),
+        ];
+    }
+
+    /**
+     * get additional data from the magento instance
+     *
+     * @return array|mixed[]
```

Comment:
what will it be then? string[]|int[] ? array and mixed are too ambiguous, hope we can provide more clarity here as to what to expect.